docs/pages/docs/query/graphql.mdx

```diff
@@ -12,14 +12,42 @@ import { Callout } from "nextra/components";
   [documentation](https://graphql.org/learn/) for an overview.
 </Callout>
 
-Ponder automatically generates a GraphQL schema based on your `ponder.schema.ts` file that uses efficient resolver logic to avoid common performance pitfalls.
+Ponder has built-in support for generating a GraphQL schema based on your `ponder.schema.ts` file that uses efficient resolver logic to avoid common performance pitfalls.
+
+## Register GraphQL middleware
+
+<Callout type="warning">
+  Starting from version `0.9.0`, the GraphQL API is not served by default and
+  must be defined in an [API function](/docs/query/api-functions).
+</Callout>
+
+To use the standard GraphQL API, register the `graphql` middleware exported from `ponder`.
+
+```ts filename="src/api/index.ts" {4,8-9}
+import { db } from "ponder:api";
+import schema from "ponder:schema";
+import { Hono } from "hono";
+import { graphql } from "ponder";
+
+const app = new Hono();
+
+app.use("/", graphql({ db, schema }));
+app.use("/graphql", graphql({ db, schema }));
```

Comment:
This may actually cause issues (instantiating the middleware twice)?

Reply:
I'm not sure, we've been doing this for a while, haven't seen any issues